extensions/amp-viqeo-player/0.1/amp-viqeo-player.js

```diff
@@ -188,6 +191,20 @@ class AmpViqeoPlayer extends AMP.BaseElement {
     return true; // Call layoutCallback again.
   }
 
+  /** @override */
+  createPlaceholderCallback() {
+    const placeholder = this.win.document.createElement('amp-img');
+    this.propagateAttributes(['aria-label'], placeholder);
```

Comment:
instead of setting aria label and alt separately, I suggest:

```
    if (placeholder.hasAttribute('aria-label')) {
      placeholder.setAttribute('alt',
          'Loading video - ' + placeholder.getAttribute('aria-label')
      );
    } else {
      placeholder.setAttribute('alt', 'Loading video');
    }
```